### PR TITLE
[Enhancement] Improve the docstring of PaviLoggerHook

### PR DESCRIPTION
**Pull request by EvelynWang-0423**
## Motivation

This PR adds new optional parameter configuration description of summarywriter to MMCV PAVI hooks.

## Modification

- Update mmcv/runner/hooks/logger/pavi.py

### CODE REVIEW

File: mmcv/runner/hooks/logger/pavi.py [hunk 1]
```diff
@@ -19,7 +19,22 @@ class PaviLoggerHook(LoggerHook):
     """Class to visual model, log metrics (for internal use).
 
     Args:
-        init_kwargs (dict): A dict contains the initialization keys.
+        init_kwargs (dict): A dict contains the initialization keys as below:
+
+            - name (str, optional): custom training name. Defaults to None,
+                which means Current work_dir.
```
Comment (AronLin) [inline mmcv/runner/hooks/logger/pavi.py:25]: Current -> current

Comment (EvelynWang-0423) [reply mmcv/runner/hooks/logger/pavi.py]: > Current -> current

done~

**Review by AronLin — APPROVED** [review]
LGTM

**Comment by zhouzaida** [comment]
https://github.com/open-mmlab/mmcv/blob/1f2500102834a01b86bf9ae4db227cd8d724fa6e/mmcv/runner/hooks/logger/pavi.py#L62-L86

Need to modify below three lines

- `self.init_kwargs['name'] = self.run_name` -> `self.init_kwargs.setdefault('name', self.run_name)`
- `self.init_kwargs['model'] = runner._model_name` -> `self.init_kwargs.setdefault('model', runner._model_name)`
- `self.init_kwargs['session_text'] = session_text` -> `self.init_kwargs.setdefault('session_text', session_text)`

File: mmcv/runner/hooks/logger/pavi.py [hunk 2]
```diff
@@ -19,7 +19,22 @@ class PaviLoggerHook(LoggerHook):
...
+                model.
+            - session_text (str, optional): Session string in YAML format.
+                Defaults to current config.
+            - training_id (int, optional): Training ID in PAVI, if you want to
+                use an existing training. Defaults to None.
+            - compare_id (int, optional): Compare ID in PAVI, if you want to
+                add the task to an existing compare. Defaults to None.
+            - overwrite_last_training (bool, optional): Whether to upload data
+                to the training with the same name in the same project, rather
+                than creating a new one. Defaults to False.
```
Comment (zhouzaida) [inline mmcv/runner/hooks/logger/pavi.py:37]:
```suggestion
            - name (str, optional): Custom training name. Defaults to None,
              which means current work_dir.
            - project (str, optional): Project name. Defaults to "default".
            - model (str, optional): Training model name. Defaults to current
              model.
            - session_text (str, optional): Session string in YAML format.
              Defaults to current config.
            - training_id (int, optional): Training ID in PAVI, if you want to
              use an existing training. Defaults to None.
            - compare_id (int, optional): Compare ID in PAVI, if you want to
              add the task to an existing compare. Defaults to None.
            - overwrite_last_training (bool, optional): Whether to upload data
              to the training with the same name in the same project, rather
              than creating a new one. Defaults to False.
```

**Review by zhouzaida — APPROVED** [review]

**Pull request merged**